tests/pass-dep/tempfile.rs

```diff
@@ -19,3 +20,24 @@ fn test_tempfile_in() {
     let dir_path = utils::tmp();
     tempfile::tempfile_in(dir_path).unwrap();
 }
+
+fn test_tempfile_ext() -> std::io::Result<()> {
```

Comment:
Why is the only test for this here? The point of this file is to test the `tempfile` crate.

Please add a test in tests/pass/shims/fs.rs. I don't think it makes sense to also have the test here -- is there anything tempfile-specific going on?

Reply:
I didn't know about `utils::prepare`. I will move the test to `fs.rs` a bit later.